nomad/structs/structs.go

```diff
@@ -8153,6 +8153,10 @@ const (
 
 	// TaskClientReconnected indicates that the client running the task disconnected.
 	TaskClientReconnected = "Reconnected"
+
+	// TaskWaitingShuttingDownDelay indicates that the task is waiting for
+	// shutdown delay before being TaskKilled
+	TaskWaitingShuttingDownDelay = "Task waiting for shutdown delay"
```

Comment:
If we look at the output of these, this name is a little wide and pushes everything to the right:

```
Recent Events:
Time                       Type                             Description
2022-10-03T14:25:34-04:00  Task waiting for shutdown delay  Waiting for shutdown delay before killing the task.
2022-10-03T14:25:34-04:00  Killing                          Sent interrupt. Waiting 5s before force killing
2022-10-03T14:25:15-04:00  Started                          Task started by client
2022-10-03T14:25:15-04:00  Task Setup                       Building Task Directory
2022-10-03T14:25:14-04:00  Received                         Task received by client
```

<img width="775" alt="Screen Shot 2022-10-03 at 2 38 26 PM" src="https://user-images.githubusercontent.com/1409219/193653527-8fe279db-f0ab-4da5-9259-932ee33debcc.png">

The event description can have more information about how we're waiting specifically on the `shutdown_delay` field and for how long, so for this event title, what if we trimmed it down to:

```suggestion
	TaskWaitingShuttingDownDelay = "Waiting for shutdown"
```

That also gives us some wiggle room to add new "waiting for shutdown" events with different sources/descriptions in the future.